.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: "integration-test"
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go-version: ["1.14", "1.13"]
+    name: Test with Go version ${{ matrix.go-version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-go@v1
+        with:
+          go-version: ${{ matrix.go-version }}
+      - run: go mod download
+      - run: go test ./...
+      - run: go run ./integration_test
+        env:
+          SECRET_KEY: xnd_development_chBDpzjQZEI8nncBLrWtwau5r7rgiuunCJ4JCqsd68rXemDd74BnRpdO0bIZAMK
```

Comment:
Can this key stored not in the `.yml` file?

Reply:
can but it would require admin to add the secret, too much of a hassle imo since this is just a development key

Reply:
Alright